.github/workflows/linux-tests.yml

```diff
@@ -12,45 +12,54 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-20.04, ubuntu-22.04 ]
+        os: [ ubuntu-20.04, ubuntu-22.04, ubuntu-24.04 ]
+        qt: [ qt5, qt6 ]
+        exclude:
```

Comment:
Why do we exclude these? A comment explaining why and if applicable when it can be turned back on would be great